docs/resources/Guild.md

```diff
@@ -890,13 +892,14 @@ Modify a guild role. Requires the `MANAGE_ROLES` permission. Returns the updated
 
 ###### JSON Params
 
-| Field       | Type    | Description                                                    |
-| ----------- | ------- | -------------------------------------------------------------- |
-| name        | string  | name of the role                                               |
-| permissions | string  | bitwise value of the enabled/disabled permissions              |
-| color       | integer | RGB color value                                                |
-| hoist       | boolean | whether the role should be displayed separately in the sidebar |
-| mentionable | boolean | whether the role should be mentionable                         |
+| Field       | Type                                     | Description                                                         |
+| ----------- | ------- | ---------------------------------------------------------------------------------------------------- |
+| name        | string  | name of the role                                                                                     |
+| permissions | string  | bitwise value of the enabled/disabled permissions                                                    |
+| color       | integer | RGB color value                                                                                      |
+| hoist       | boolean | whether the role should be displayed separately in the sidebar                                       |
+| icon        | string  | the role's [icon hash](#DOCS_REFERENCE/image-formatting) (if the guild has the `ROLE_ICONS` feature) |
+| mentionable | boolean | whether the role should be mentionable                                                               |
```

Comment:
```suggestion
| Field       | Type                                     | Description                                                             |
| ----------- | ---------------------------------------- | ----------------------------------------------------------------------- |
| name        | string                                   | name of the role                                                        |
| permissions | string                                   | bitwise value of the enabled/disabled permissions                       |
| color       | integer                                  | RGB color value                                                         |
| hoist       | boolean                                  | whether the role should be displayed separately in the sidebar          |
| icon        | [image data](#DOCS_REFERENCE/image-data) | the 128x128 role icon image (if the guild has the `ROLE_ICONS` feature) |
| mentionable | boolean                                  | whether the role should be mentionable                                  |
```

Reply:
That collidates with https://github.com/discord/discord-api-docs/pull/3815#pullrequestreview-752606768

Reply:
@advaith1 what u think?

Reply:
I'm pretty sure they have suggested changes in the wrong place. As advaith mentioned, image data is what the client sents.

Reply:
well, I implemented it into my lib like it's actually noted and some users with access to role_icons tested it and it worked.
I'm unsure

Reply:
Same as above, this should not say 128x128 and it should also support emojis

Reply:
I'm not sure why this is marked as resolved, it's still incorrect afaik. I'm not sure what you mean by your above comment, don't see how one can get an icon hash before having created a role @Lulalaby

Reply:
Could you please redo a new review?
I'm a bit confused rn about the unicode emoji.

Reply:
Sure, unicode emoji is presumably only for unicode emojis, for guild emojis, the client just re-uploads the image as an icon.